client/src/components/buttons/Button.js

```diff
@@ -36,41 +38,44 @@ const Basicbutton = styled.button`
   border-radius: 15px;
   font-size: 1rem;
   cursor: pointer;
+  flex-grow: 1;
   font-family: 'Source Code Pro', monospace;
 `;
 
 // Styled button applies to yes/no
 const Styledbutton = styled(Basicbutton)`
   margin: 5%;
-  margin-right: 0%;
   padding-top: 3px;
   padding-bottom: 4px;
-  width: 5rem;
+  min-width: 5rem;
+  flex-grow: 1;
 `;
 
 // Applies to other buttons
 const Multiplebutton = styled(Basicbutton)`
   margin: 0.5%;
+  height: 6rem;
```

Comment:
Maybe slightly too high? All the buttons cover the screen here so the user has to scroll.

Is the idea to have all buttons the same height, and because one has two lines of text all are this big?

![button-scroll](https://user-images.githubusercontent.com/25408167/38669150-8c1116fa-3e3d-11e8-980a-4c02ee9bad4f.gif)